examples/nuxt-3/queries/collection-page-query.js

```diff
@@ -0,0 +1,51 @@
+import CollectionProductFragment from './collection-product-fragment.query';
+
+export default `
+  query CollectionPage($handle: String!) {
+    allProductCollections(filter: { handles: [$handle] }) {
+      edges {
+        node {
+          nacelleEntryId
+          sourceEntryId
+          content {
+            title
+          }
+          productConnection(first: 12) {
+            pageInfo {
+              hasNextPage
+              endCursor
+            }
+            totalCount
+            edges {
+              node {
+                ...CollectionProductFragment
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  ${CollectionProductFragment}
+`;
+
+export const CollectionProductsQuery = `
+    query allProductCollections(filter: {handles: [$handle]}) {
+        edges {
+            node {
+                productConnection(first: 12, after: $after) {
+                    pageInfo {
+                        hasNextPage
+                        endCursor
+                    }
+                    edges {
+                        node {
+                            ...CollectionProductFragment
+                        }
+                    }
+                }
+            }
+        }
+    }
+    ${CollectionProductFragment}
+`;
```

Comment:
Might be nice on the eyes to use consistent indentation with these queries.

Reply:
Oops. Looks like this was actually malformed gql for the second query. Fixed